packages/sdk-client/src/domain/domain-interface.ts

```diff
@@ -81,53 +83,34 @@ export enum SmsRegion {
   AUSTRALIA = 'au'
 }
 
-export function getRegion(value: string | undefined): SmsRegion | undefined {
-  if (!value) {
-    return undefined;
-  }
-
-  for (const region of Object.values(SmsRegion)) {
-    if (region === value.toLowerCase()) {
-      return region as SmsRegion;
-    }
-  }
-  console.error(`No region exist for the value '${value}'`);
-  return undefined;
-}
+export type SmsRegionFlexible = SmsRegion | string;
```

Comment:
By inverting the point of view; could the type being more user friendly ?
I mean:
```javascript
export enum SupportedSmsRegion {
  UNITED_STATES = 'us',
  EUROPE = 'eu',
  BRAZIL = 'br',
  CANADA = 'ca',
  AUSTRALIA = 'au'
}
export type SmsRegion = SupportedSmsRegion | string;
```
So when function prototype or variable usage will reference the type, it will be named `SmsRegion`.
Seems to be more "user friendly" than a `SmsRegionFlexible` reference and usage.
What do you think?

Reply:
This would be more understandable indeed. The issue being that with this implementation, I can't do SmsRegion.EUROPE as SmsRegion refers to a Type, but would be used as a value
A solution would be to trick the user and define on top of the enum a new object SmsRegion such as 
```
export const SmsRegion = {
  ...SupportedSmsRegion,
};
```
WDYT?

Reply:
As usual, we should have end user and SDK usage in mind first.
so if a `SmsRegion` type is identified as  the best way to reach this objective, if by implementing it as you described in your comment above: it is **very** fine
Then, using same pattern for Conversation, if SDK usage enable to store a variable/use a parameter type named `ConversationRegion` and reference `ConversationRegion.UNITED_STATES` or use `us` directly : it should provide a better DX (IMO)

Reply:
Ok, we can do that. Keep in mind the trick I mentioned above: ConversationRegion.UNITED_STATES will not reference the enum, but an object of which we read the value associated to the key `UNITED_STATES` 😉

Reply:
Fixed